src/signals/IncidentMap/components/utils/compute-nr-of-incidents-per-filter.ts

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import type { Feature } from 'geojson'
+
+import type { Filter, Properties } from '../../types'
+
+export const computeNrOfIncidentsPerFilter = (
```

Comment:
Name is not very clear for me. Maybe something like: `getFiltersWithNrOfIncidents` ?

Reply:
getFiltersWithincidentCounts? we gebruiken nu count op een paar plekken